NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -203,28 +205,28 @@ class NeuroID private constructor(
         return clientID
     }
 
-    fun getForceStart(): Boolean? {
+    internal fun getForceStart(): Boolean? {
         return forceStart
     }
 
-    fun setForceStart(activity: Activity) {
+    fun registerPageTargets(activity: Activity) {
         this.forceStart = true
         NIDActivityCallbacks().forceStart(activity)
     }
 
-    fun getTabId(): String = NIDServiceTracker.rndmId
+    internal fun getTabId(): String = NIDServiceTracker.rndmId
 
-    fun getFirstTS(): Long = timestamp
+    internal fun getFirstTS(): Long = timestamp
 
-    fun getJsonPayLoad(context: Context): String {
+    internal fun getJsonPayLoad(context: Context): String {
         return getDataStoreInstance().getJsonPayload(context)
     }
 
-    fun resetJsonPayLoad() {
+    internal fun resetJsonPayLoad() {
```

Comment:
We are using this in integration testing. Cannot make this one internal yet. Have to fix the test in our app first.

Reply:
removed from test, test fixed.